.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,4 +4,6 @@
 
 *Test Results:*
 
+**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
```

Comment:
Can you add the hyperlink to helm readme. Someone may by mistake update the main readme. Better to be safe

Reply:
Resolved.